Source/Concepts/ConceptAs.cs

```diff
@@ -51,54 +51,39 @@ public class ConceptAs<T> : IEquatable<ConceptAs<T>>, IComparable<ConceptAs<T>>,
         /// <param name="left">Left <see cref="ConceptAs{T}"/>.</param>
         /// <param name="right">Right <see cref="ConceptAs{T}"/>.</param>
         /// <returns>true if the left <see cref="ConceptAs{T}"/> is equal to the right <see cref="ConceptAs{T}"/>; otherwise, false.</returns>
-        public static bool operator !=(ConceptAs<T> left, ConceptAs<T> right)
-        {
-            return !(left == right);
-        }
+        public static bool operator !=(ConceptAs<T> left, ConceptAs<T> right) => !(left == right);
 
         /// <summary>
         /// Larger than comparison operator for comparing two <see cref="ConceptAs{T}"/>.
         /// </summary>
         /// <param name="left">Left <see cref="ConceptAs{T}"/>.</param>
         /// <param name="right">Right <see cref="ConceptAs{T}"/>.</param>
         /// <returns>true if the left <see cref="ConceptAs{T}"/> is larger than the right <see cref="ConceptAs{T}"/>; otherwise, false.</returns>
-        public static bool operator >(ConceptAs<T> left, ConceptAs<T> right)
-        {
-            return left.CompareTo(right) == 1;
-        }
+        public static bool operator >(ConceptAs<T> left, ConceptAs<T> right) => left.CompareTo(right) == 1;
 
         /// <summary>
         /// Smaller than comparison operator for comparing two <see cref="ConceptAs{T}"/>.
         /// </summary>
         /// <param name="left">Left <see cref="ConceptAs{T}"/>.</param>
         /// <param name="right">Right <see cref="ConceptAs{T}"/>.</param>
         /// <returns>true if the left <see cref="ConceptAs{T}"/> is smaller than the right <see cref="ConceptAs{T}"/>; otherwise, false.</returns>
-        public static bool operator <(ConceptAs<T> left, ConceptAs<T> right)
-        {
-            return left.CompareTo(right) == -1;
-        }
+        public static bool operator <(ConceptAs<T> left, ConceptAs<T> right) => left.CompareTo(right) == -1;
 
         /// <summary>
         /// Larger than or equal comparison operator for comparing two <see cref="ConceptAs{T}"/>.
         /// </summary>
         /// <param name="left">Left <see cref="ConceptAs{T}"/>.</param>
         /// <param name="right">Right <see cref="ConceptAs{T}"/>.</param>
         /// <returns>true if the left <see cref="ConceptAs{T}"/> is larger or equal than the right <see cref="ConceptAs{T}"/>; otherwise, false.</returns>
-        public static bool operator >=(ConceptAs<T> left, ConceptAs<T> right)
-        {
-            return left.CompareTo(right) > -1;
-        }
+        public static bool operator >=(ConceptAs<T> left, ConceptAs<T> right) => left.CompareTo(right) > -1;
```

Comment:
This doesn't matter - but I just like the use of 0 better.
```suggestion
        public static bool operator >=(ConceptAs<T> left, ConceptAs<T> right) => left.CompareTo(right) >= 0;
```